client-js/app/elements/labrad-manager.ts

```diff
@@ -1,9 +1,14 @@
-import {ConnectionInfo} from '../scripts/manager';
+import {ConnectionInfo, ManagerApi} from '../scripts/manager';
 
 @component('labrad-manager')
 export class LabradManager extends polymer.Base {
 
   @property({type: Array, notify: true})
   connections: Array<ConnectionInfo>;
 
+  mgr: ManagerApi;
+
+  closeConnection(event) {
+    this.mgr.connectionClose(event.currentTarget.connId);
```

Comment:
It would be good to ask for confirmation before actually closing the connection. Can just use `window.confirm` for simplicity. This is not an operation we will do often, and we should be careful to avoid killing connections accidentally.